static-site/src/sections/community-page.jsx

```diff
@@ -63,11 +94,45 @@ class Index extends React.Component {
           </splashStyles.CenteredFocusParagraph>
         </FlexCenter>
         <HugeSpacer />
-        <Cards cards={communityCards}/>
         <HugeSpacer />
+        {this.state.loaded && <DatasetSelect
+          title="Search Community Datasets and Narratives "
+          datasets={tableData}
+          columns={tableColumns}
+          rowSort={[]}
+        />}
       </GenericPage>
     );
   }
 }
 
 export default Index;
+
+
+function parseTableData(yamlData) {
+  const communityUrlPattern = new RegExp("^/community(?<narrative>/narratives)?/(?<org>[^/]+)/(?<repo>[^/]+)");
+
+  return yamlData.data
+    .map((entry) => {
+      const url = new URL(entry.url, "https://nextstrain.org");
+      const urlMatches = url.pathname.match(communityUrlPattern);
+
+      if (!urlMatches) {
+        console.warn(`Removing data entry "${entry.name}" as URL <${entry.url}> is not valid`);
+        return undefined;
```

Comment:
When testing 9095e41, I noticed that this error-handling here (which I preserved) doesn't make a very loud noise during `gatsby build`. @jameshadfield Thoughts on throwing an exception here instead? If we were requesting the YAML file on every page load, then a warning + skip on the client would make sense to me as the YAML would change more independently, but since this code runs at static site build time, an exception will help ensure we keep the data correct.

Reply:
The code runs both at compile time and on the client each page load. At least, it does in my testing, and this is how I understand Gatsby to work. 

I _think_ you could condition on this via something like 👇  if you want to pursue this?

```js
if (typeof window === 'undefined') {
  throw new Error(...)
} else {
  console.warn(...);
}
```

Reply:
Ah, it does appear to run at both build and run time, despite the latter being completely unnecessary. I thought it was supposed to be compiled away given that it's all static and cannot change.

In any case, I'm not going to pursue this more. Will hopefully just ditch Gatsby sooner than later.